app/views/admin/applications/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<% @application.application_pets.each do |app_pet| %>
+  <div id="AppPet-<%= app_pet.pet_id %>">
+  <% app_pet.pet.name %>
```

Comment:
The reason that your pet name is not showing up in the view is because you need to add the `=` to make this a render erb tag.